ProcessMaker/Cache/Screens/ScreenCacheManager.php

```diff
@@ -147,18 +150,19 @@ public function missing(string $key): bool
      * @param string $language Language code
      * @return bool
      */
-    public function invalidate(int $screenId, string $language): bool
+    public function invalidate($params): void
     {
         // Get all keys from cache that match the pattern for this screen ID
         // TODO Improve this to avoid scanning the entire cache
```

Comment:
@rodriquelca is this TODO fixed by `$keys = $this->cacheManager->get($pattern);`?
Or need still to improve more this function?